src/autocomplete.js

```diff
@@ -508,13 +508,14 @@ export default class Autocomplete extends Component {
     }
 
     const computedMenuAttributes = {
-      // Copy the attributes passed as props
-      ...menuAttributes,
       // And add the values computed for the autocomplete
+      ...menuAttributes,
       id: `${id}__listbox`,
       role: 'listbox',
+      'aria-labelledby': id,
```

Comment:
**issue** This would be a breaking change for people who may have set the ID to something outside the component. Unlike `id` and `role`, which we need to be in control of, we can happily let people use anything else to label the menu. It's a cheap change as we only need to re-order the properties of the object to avoid the breaking change while setting a sensible default that automatically labels the menu:

```js
{
  'aria-labelledby': id,
  ...menuAttributes,
  id: `${id}__listbox`,
  /* ... */
}